File: practices/time.py
# SB 6th Time of day 

import datetime

time = int(input("What time is it in military time?"))

if time <= 11:
    print("Good morning!")
elif time <= 17:
    print("Good afternoon!")
elif time <= 24:
    print("Good evening!")
else:
    print("Thats more hours that exist in a day")

File: c_programming/notes/time.c
// SB 6th time of day
#include <stdio.h>
#include <string.h>

int main(void){
    int time;
    printf("What time is it in military time?: ");
    scanf("%d", &time);

    if(time <= 11){
        printf("Good morning!\n");
    }else if(time <= 17){
        printf("Good afternoon!");
    }else if(time <= 24){
        printf("Good evening!");
    }else{
        printf("Thats more hours that exist in a day");
    }
    return 0;
}